src/main/java/com/sap/sgs/phosphor/fosstars/data/github/GitHubDataFetcher.java

```diff
@@ -101,7 +101,7 @@
   /**
    * A cache of local repositories.
    */
-  final LRUMap<GitHubProject, LocalRepository> localRepositories
+  static final LRUMap<GitHubProject, LocalRepository> localRepositories
       = new LRUMap<>(LOCAL_REPOSITORIES_CACHE_CAPACITY, SCAN_UNTIL_REMOVABLE);
```

Comment:
@mibo Did you want to be able to set capacity of this cache?

Reply:
No, the cache/map I mentioned was the [entries](https://github.com/SAP/fosstars-rating-core/blob/master/src/main/java/com/sap/sgs/phosphor/fosstars/data/github/GitHubDataCache.java#L35) map in the `GitHubDataCache`.
As the default and not changeable size of [5](https://github.com/SAP/fosstars-rating-core/blob/master/src/main/java/com/sap/sgs/phosphor/fosstars/data/github/GitHubDataCache.java#L25) looks a bit small.
With the sixth request (via `GitHubDataFetcher#repositoryFor`) the first cached entry is already removed (if only one instance of the `GitHubDataFetcher` is used).
In case the single `GitHubDataFetcher` is used in multiple threads this may lead to the case that the cached entries are removed before they get requested a second time (which make the cache a bit useless).

Personally I like if such cache sizes are configurable so that whoever use it can adjust it to their needs.

Reply:
Now the `localRepositories` is also made `static` and its usages are added within `synchronized` block. This should avoid multiple thread clearing the cache unknowingly.